server/utils/utils.js

```diff
@@ -42,6 +43,14 @@ const properCaseFullName = name =>
         .map(properCaseName)
         .join(' ')
 
+const stringToHash = inputString => {
+  return crypto.createHash('sha256').update(inputString).digest('base64')
+}
+
+const isHashedString = (hashValue, originalString) => {
+  return hashValue === crypto.createHash('sha256').update(originalString).digest('base64')
```

Comment:
might be better to use stringToHash? 

missing type signatures too on these.

Reply:
changed module from js to ts
above changes made